tests/unit/test_data_utils.py

```diff
@@ -0,0 +1,144 @@
+import pandas as pd
+import pytest
+
+from pseudopeople.constants import paths
+from pseudopeople.constants.metadata import DatasetNames
+from pseudopeople.loader import (
+    combine_joint_filers,
+    flatten_data,
+    load_and_prep_1040_data,
+)
+from pseudopeople.schema_entities import COLUMNS, DATASETS
+
+
+@pytest.fixture(scope="module")
+def dummy_1040():
```

Comment:
Looks good. I have a few suggestions. 
- You should have some filers who are not reference persons. 
- You should have some spouses who are not filing jointly
- You should have some reference persons who don't have a spouse